ckanext/dgu/bin/fix_mandate.py

```diff
@@ -0,0 +1,62 @@
+'''
+Fix the secondary theme to be a JSON list (from list of Python unicode strings)
+
+e.g.
+
+[u'Crime'] -> ['Crime']
+'''
+
+import sys
+import common
+import json
+from optparse import OptionParser
+from ckan import model
+
+from running_stats import StatsList
+
+stats = StatsList()
+
+class FixMandate(object):
+    @classmethod
+    def command(cls, config_ini, write):
+        common.load_config(config_ini)
+        common.register_translator()
+
+        rev = model.repo.new_revision()
+        rev.author = 'fix_mandate.py'
```

Comment:
Can you change this to "script-fix_mandate.py" so that the changes don't appear in the publisher activity report?
See: https://github.com/datagovuk/ckanext-dgu/pull/80/files#diff-4d9c6deac41b2c7fd667263a45e84626R291 for why "script-" is needed